api/types/stats.go

```diff
@@ -170,6 +170,9 @@ type Stats struct {
 type StatsJSON struct {
 	Stats
 
+	Name string `json:"name,omitempty"`
+	ID   string `json:"id,omitempty"`
```

Comment:
Mainly because I wanted to keep `Stats` struct with only metrics field and no metadata.